### PR TITLE
Update relative time tags and bid info on homepage

### PR DESCRIPTION
**Pull request by jessieay**
* Part of https://github.com/18F/micropurchase/issues/949

### CODE REVIEW

File: spec/presenters/status_presenter/available_spec.rb [hunk 1]
```diff
@@ -1,29 +1,40 @@
...
-        ended_at: Time.now + 2.days,
-        start_price: 3500
-      )
-      create(:bid, auction: a, amount: 3000)
-      a
-    end
-
+  describe '#tag_data_value_status' do
     it "has a twitter status data value with human readable time expression" do
+        presenter = StatusPresenter::Available.new(auction)
```
Comment (adelevie) [inline spec/presenters/status_presenter/available_spec.rb]: spacing

File: spec/presenters/status_presenter/available_spec.rb [hunk 2]
```diff
@@ -1,29 +1,40 @@
...
       expect(presenter.tag_data_value_status).to eq("2 days left")
     end
+  end
 
+  describe '#tag_data_label_2' do
     it "has a twitter second label that indicates bidding is open" do
+        presenter = StatusPresenter::Available.new(auction)
```
Comment (adelevie) [inline spec/presenters/status_presenter/available_spec.rb]: spacing

File: spec/presenters/status_presenter/available_spec.rb (hunk 1 above)
Comment (jessieay) [inline spec/presenters/status_presenter/available_spec.rb]: fixed at  99f02a4 -- nice eagle 👀

**Comment by adelevie** [comment]
LGTM

**Comment by jessieay** [comment]
cool, feel free to merge @adelevie

**Pull request merged**